sky/tools/gn

```diff
@@ -53,7 +53,9 @@ def to_gn_args(args):
         gn_args['target_os'] = 'ios'
         gn_args['ios_deployment_target'] = '7.0'
         gn_args['use_ios_simulator'] = args.simulator
-        if not args.simulator:
+        if args.simulator:
+            gn_args['use_libjpeg_turbo'] = False
```

Comment:
Yeah, the stock libjpeg was able to decode images fine on the simulator. Haven't pull this merge yet. Will check again.